metricflow-semantics/metricflow_semantics/test_helpers/semantic_manifest_yamls/simple_manifest/metrics.yaml

```diff
@@ -11,7 +11,10 @@ metric:
   description: "average booking value metric"
   type: simple
   type_params:
-    measure: average_booking_value
+    measure:
+      name: average_booking_value
+      filter: "average_booking_value > 10"
```

Comment:
Thank you for adding! So silly that we didn't have this before

Reply:
woops, i misadded this one actually. I added a new metric that has a measure filter below.